apps/marketing/src/app/(marketing)/singleplayer/page.tsx

```diff
@@ -1,5 +1,11 @@
+import type { Metadata } from 'next';
+
 import { SinglePlayerClient } from './client';
 
+export const metadata: Metadata = {
+  title: 'Single Player Mode | Documenso',
```

Comment:
Could you replace this with `Singleplayer`?

Also you can put the template into the root layout